BUILD

```diff
@@ -0,0 +1,31 @@
+package(default_visibility = ["//visibility:public"])
+load("@subpar//:subpar.bzl", "par_binary")
+load("@my_deps//:requirements.bzl", "requirement")
+
+par_binary (
+    name="rss_downloader",
+    srcs=[
+        'rss_downloader.py'
+    ],
+    deps = [
+      requirement("appdirs"),
+      requirement("feedparser"),
+      requirement("packaging"),
+      requirement("pyparsing"),
+      requirement("requests"),
+      requirement("schedule"),
+      requirement("six")
+    ]
+)
+
+py_test(
+    name='test_rss_downloader',
+    srcs=[
+        'test_cases.py'
+    ],
+    data = [ 'test_responses' ],
```

Comment:
is there a data file called `'test_responses'`?

Reply:
yes. A data folder called test_responses contains mock RSS feed responses. During testing, an http server is spun up and then configured to use the test_responses.